src/persist-client/ci/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+MZFROM ubuntu-base
+
+RUN apt-get update && TZ=UTC DEBIAN_FRONTEND=noninteractive apt-get install -y --no-install-recommends \
+    bzip2 \
+    curl \
+    git \
+    gnuplot \
+    openjdk-11-jre # WIP is this right?
```

Comment:
I can't use openjdk-8-jre anymore, apparently. the 0.2.1 release of maelstrom is compiled with something too new. any opinions on openjdk vs I dunno something else?

Reply:
openjdk-11-jre is exactly right!